cddl/measurement-map.cddl

```diff
@@ -1,5 +1,4 @@
 measurement-map = {
-  ? &(mkey: 0) => $measured-element-type-choice
-  &(mval: 1) => measurement-values-map
-  ? &(authorized-by: 2) => [ + $crypto-key-type-choice ]
+  ? "__authorities" => [ + $crypto-key-type-choice ]
```

Comment:
Why does this change need to be part of this changelist? Changing an int to a string will make comid files bigger, can we discuss that separately?

Reply:
Authorities are more predicate than they are object, so I'm happy to move them into the triple / stateful-environment-map. I'd like to ask the group about it though.

Reply:
The "__"-prefix are intended to be a placeholder (not a magic string). Personally, I think in the context of this map, negative values are perfectly fine.

Reply:
I've changed the placeholder string to a &(fixme-authority: -1) codepoint since I plan to follow up with another PR that lifts authorities out of measurement-map and into the surrounding uses when appropriate.